.github/workflows/emulator.yml

```diff
@@ -0,0 +1,22 @@
+name: Emulator-CI
+
+on: [push]
```

Comment:
Is this executed when any push happens for other directories?
For example, when the explorer is updated, this CI for the emulator will be executed.
I think it is better to use `paths`.
https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths

Reply:
Thank you for review comment.
Workflow modified based on the above comment.